### DISCUSSION TITLE
PR? Issue? Idea? Q&A! Show!

### DISCUSSION THREAD

**Discussion by naradesign** [naradesign/css-reset-by-class#1] [General]
* PR(Pull Requests)을 보내거나
* 문제를 공유하거나(Issues) 
* 아이디어를 제안하거나(Discussions - Ideas)
* 질문을 던져 보세요(Discussions - Q&A)
* 직접 만든 CSS 리셋을 공유해 주세요.(Discussions - Show and tell)

**Comment by celinaDev** [comment]
안녕하세요. 유익한 수업내용 감사드립니다.
한 가지 질문이 있는데요.

한, 영, 중, 일을 지원하는 프로덕트의 경우 아래와 같이 초기화 해도 무관할까요?
```
body {
  margin: 0;
}

:lang(en) { overflow-wrap: break-word; }
```

**Comment by ui-kimjisu** [comment]
패스트 캠퍼스 온라인 강의 과제로 들어와봤어요.
선뜻 나눠주시는 경험에 오랫동안 도움 받고 있어요.
고맙습니다. 


:where(button[class], dialog[class], input[class], mark[class], meter[class], progress[class]) {
    background-color: transparent;
}

버튼이나 인풋요소에 배경색을 적용할 때가 많아서 저는 제외하고 사용할 것 같아요.
